### ISSUE TITLE
1,4,11: Minor fixes on the ports configuration

### ISSUE THREAD

**Issue by nunoguedelha** [ami-iit/yarp-openmct#65]
- [x] 1: The force visualization connects to the legs' ports, instead of the ones of the feet
- [x] 4: The Yarp port names in the script configuration have the hardcoded prefix `icubSim`
- [x] 11: Telemetry and Visualization Servers log "localhost" instead of the raw IP address on the output stream.
- [x] Update README.md.

**Comment by nunoguedelha** [comment]
## 1: The force visualization connects to the legs' ports, instead of the ones of the feet

Replaced visualization of EE wrench on Hip by the one on the foot:
- Replaced the selected port `yarpName` in the `portInConfig` table.
- Renamed the telemetry signal in the `ICubTelemetry.state` definition, update, and respective `ICubTelemetry.generateTelemetry` and OpenMct dictionary entries.

**Comment by nunoguedelha** [comment]
## 4: The Yarp port names in the script configuration have the hardcoded prefix icubSim

- Move the whole port configuration to a separate configuration file `default.json` in a  new configuration folder `config`.
- `default.json` holds: the ports configuration `portInConfig`; the servers configurations `telemVizServer`, `consoleServer` and `openmctStaticServer`, with the 'host' and 'port' fields; the Yarp ports prefix `robotYarpPortPrefix` (`/icub`, `/icubSim`).
- `default.json` has a JSON format but its format was extended to support the use of variables in the JSON literals:
  ```js
  {
      "robotYarpPortPrefix": "/icubSim",
      ...
      "portInConfig": {
          "sens.imu": {
              "yarpName": "${this.robotYarpPortPrefix}/inertial",
              ...
          },
          ...
      }
  }
  ```
- The config file is processed in `./config/processedDefault.js`, by using string interpolation in Template literals [[1]], and the `eval` function [[2]]. A traversal function applies this processing to each <value> in the nested <key:value> pairs of the "config" JSON object.

[1]: https://www.w3schools.com/js/js_string_templates.asp
[2]: https://www.w3schools.com/jsref/jsref_eval.asp

**Comment by nunoguedelha** [comment]
## 11: Telemetry and Visualization Servers log "localhost" instead of the raw IP address on the output stream

This was fixed by using directly the address and port bound to the `net.Server` class object returned by the server creation and launch:
```js
const telemServer = app.listen(<port>, <host>, function () {...});
```

#### References
https://nodejs.org/api/net.html#class-netserver
https://nodejs.org/api/net.html#serveraddress

**Comment by nunoguedelha** [comment]
Updated the README. Work completed.

Implemented in #74 .

**Comment by nunoguedelha** [comment]
CC @traversaro @S-Dafarra @RiccardoGrieco
